src/objects/zcl_abapgit_object_intf.clas.testclasses.abap

```diff
@@ -79,5 +345,4 @@ CLASS ltcl_serialize IMPLEMENTATION.
       iv_json_b = lv_expected ).
     cl_abap_unit_assert=>assert_true( lv_is_equal ).
   ENDMETHOD.
-
-ENDCLASS.
+ENDCLASS.
```

Comment:
The testclass file has no empty line at the end. Maybe due to manual merging? To prevent empty diffs in abapGit, you can add an empty line.

Reply:
alternatively, pull the code using abapGit, after pull it will suggest the default abapGit formatting

Reply:
Done